app/src/main/java/com/hyperether/getgoing/db/DbHelper.java

```diff
@@ -14,9 +13,14 @@
  */
 public class DbHelper {
 
+    public interface Callback {
```

Comment:
@NikolaHadzicNs give listener some intuitive and also Android styled name, like OnDataLoaded or similar